posthog/api/test/test_capture.py

```diff
@@ -143,6 +145,39 @@ def test_capture_event(self, kafka_produce):
             self._to_arguments(kafka_produce),
         )
 
+    @patch("axes.helpers.get_cache")
```

Comment:
had to go into axes internals to figure out a good place to patch so I can test that we don't touch it 😅 

adding in the optional middleware makes this blow up

Reply:
This feels like it could be a comment on this test class to help the future traveller

Reply:
Sounds good!